icepack.setup

```diff
@@ -469,14 +482,16 @@ if !(\$?CODECOV_TOKEN) then
   endif
 endif
 
-set report_name = "${shhash}:${branch}:${machine} ${testsuite}"
-set use_curl = 1
+#for lcov
+setenv PERL5LIB ~/usr/lib/perl5/site_perl/5.18.2/x86_64-linux-thread-multi/
```

Comment:
this seems fishy to me... I guess this is needed for lcov ? 
It might be a good idea to document the lcov installation procedure somewhere (could just be in the wiki). 

I would prefer if this was a variable in the env file (I haven't checked that it could work though).

Reply:
It is fishy.  codecov doesn't require any special install and looks better, but it isn't robust.  lcov seems to be quite robust, but there are lots of other tradeoffs including installation issues as well as usage issues.  In practice, we can only produce full lcov output on cheyenne (NCAR machine) because it's the only machine we can run the pio stuff so the above is for cheyenne.  I'm still not sure whether lcov needs to be usable by a broader range of developers.  We're also writing the lcov output to a github page under my username.  I meant to take some notes about getting lcov to work and dropped that ball.  Let me see if I can recover some of that information and put it somewhere.  I'll try to test lcov on another machine.

Reply:
I created a wiki page for lcov installation.  @phil-blain, if you see anything that can be improved, please edit and let me know.  https://github.com/CICE-Consortium/About-Us/wiki/Install-lcov

Reply:
That's good. I made some minor edits for the instructions to be more shell-agnostic (I'm pretty sure it works by haven't tested it my self). 

Why is is not an option to put the `setenv PERL5LIB ~/usr/lib/perl5/site_perl/5.18.2/x86_64-linux-thread-multi/` line in `env.cheyenne_gnu` ?

Reply:
I was just going to do that, move both the setenv PATH and setenv PERL5LIB to the env file and check if that works.  That's definitely a cleaner solution.

Reply:
I have updated the implementation, so the paths are now set in the env files, and this will also hopefully mean my lcov install will work for for others on those machines.  I have confirmed this works on a couple platforms and have updated the lcov install instructions.